infrastructure/code/backend_service.go

```diff
@@ -349,3 +345,102 @@ func (s *SnykCodeHTTPClient) checkResponseCode(r *http.Response) error {
 
 	return errors.New("Unexpected response code: " + r.Status)
 }
+
+type AutofixStatus struct {
+	message string
+}
+
+func (s *SnykCodeHTTPClient) RunAutofix(
+	ctx context.Context,
+	options AutofixOptions,
+) ([]AutofixSuggestion,
+	AutofixStatus,
+	error,
+) {
+	method := "code.RunAutofix"
+	span := s.instrumentor.StartSpan(ctx, method)
+	defer s.instrumentor.Finish(span)
+
+	requestId, err := performance2.GetTraceId(span.Context())
+	if err != nil {
+		log.Err(err).Str("method", method).Msg("Failed to obtain request id. " + err.Error())
+		return nil, AutofixStatus{}, err
+	}
+	log.Debug().Str("method", method).Str("requestId", requestId).Msg("API: Retrieving analysis for bundle")
```

Comment:
maybe the comment should mention autofix :)

Reply:
✅